Exiled.API/Features/Player.cs

```diff
@@ -1872,6 +1877,16 @@ public void Broadcast(Broadcast broadcast, bool shouldClearPrevious = false)
         /// <returns>dropped <see cref="Pickup"/>.</returns>
         public Pickup DropItem(Item item) => Pickup.Get(Inventory.ServerDropItem(item.Serial));
```

Comment:
```suggestion
        public Pickup DropItem(Item item) => item is not null ? Pickup.Get(Inventory.ServerDropItem(item.Serial)) : null;
```
I know this is not directly related to your PR but would be nice